packages/expo-calendar/src/Calendar.ts

```diff
@@ -558,6 +558,149 @@ export type DaysOfTheWeek = {
 
 export { PermissionResponse, PermissionStatus, PermissionHookOptions };
 
+/**
+ * The result of presenting the calendar dialog.
+ * These constants map to [`EKEventEditViewAction`](https://developer.apple.com/documentation/eventkitui/ekeventeditviewaction) and [`EKEventViewAction`](https://developer.apple.com/documentation/eventkitui/ekeventviewaction).
+ * @platform ios
+ * */
+export type CalendarDialogResultActionsIos = 'done' | 'canceled' | 'deleted' | 'responded';
+
+/**
+ * The result of launching the calendar Activity.
+ * Android doesn't provide enough information to determine the user's action, so the result is always 'done'.
+ * The user may have cancelled the dialog, saved the event, or deleted it.
+ * @platform android
+ * */
+export type CalendarDialogResultActionsAndroid = 'done';
```

Comment:
To be honest I'm more in favor of having just one type for this, no matter what is the platform. If we document it well, then this is totally fine. The generated docs will probably get simpler too.

Reply:
The problem here is that we cannot annotate single string union values with `@platfrom` annotation which forces use to use the description to mark the differences, which IMO make the documentation a bit less clear.

Wdyt about using `enum` here instead? This should satisfy the requirement of one entity, and would allow creating a better documentation, I think.